erpnext/stock/get_item_details.py

```diff
@@ -135,22 +135,44 @@ def get_basic_details(args, item_doc):
 	user_default_warehouse_list = get_user_default_as_list('warehouse')
 	user_default_warehouse = user_default_warehouse_list[0] \
 		if len(user_default_warehouse_list)==1 else ""
+	# Modif CAF
+	argsCustomer =  {
+                doctype: "Customer",
+                name: args.customer
+            }
+	customerData = frappe.client.get(argsCustomer)
+
+	translate_description = ""
+	if item.Description_Table[0].Language == customerData.customer_language:
+		translate_description = item.Description_Table[0].Description
+	if item.Description_Table[1].Language == customerData.customer_language:
+                translate_description = item.Description_Table[1].Description
+	if item.Description_Table[2].Language == customerData.customer_language:
+                translate_description = item.Description_Table[2].Description
+	if item.Description_Table[3].Language == customerData.customer_language:
+                translate_description = item.Description_Table[3].Description
+
```

Comment:
not able to understand this, I think not all changes are updated (specially to Item)

either ways, seems why only 4 different line items for language?

this can easily be done via a query

```
frappe.db.sql("select description from `tabItem Description` where language = %s and item_code=%s", customer_language, item_code)
```

Also, we use under_score instead of camelCasing